nomenclature/processor/region.py

```diff
@@ -403,6 +408,43 @@ def _filter_dict_args(
         }
 
 
+def _aggregate_with_model_native(
+    model_df: IamDataFrame,
+    vars: Union[str, List[str]],
+    common_region: str,
+    constituent_regions: List[str],
+    **aggregation_kwargs,
+) -> IamDataFrame:
+
+    # Get all results for common_region
+    model_native = model_df.filter(region=common_region, variable=vars)
```

Comment:
Why is this called `model_native` if it is the selection of the common regions?

Reply:
I called it that way since it's results that are natively reported from the model and not obtained through region aggregation. I can use a different name though.

Reply:
I think this is rather confusing here: there are native regions and common regions, and there are originally-submitted results versus a processed IamDataFrame (including the aggregated results). I would still rather call it `common_region_df`.

Reply:
Ah ok, so using `common_region_df` for results that are (potentially) comprised of originally-submitted as well as processed results. I'll use that name and add a comment to make it more clear.